renderer/components/editor/buttons/play-pause.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {PlayIcon, PauseIcon} from '../../../vectors';
+
+const PlayPause = ({isPlaying = false, pause, play}) => <span onClick={isPlaying ? pause : play}>{isPlaying ? <PauseIcon shadow fill="#FFF" hoverFill="#FFF"/> : <PlayIcon shadow fill="#FFF" hoverFill="#FFF"/>}</span>;
```

Comment:
I think you can add `pause` as `onClick` on the PauseIcon, and `play` as `onClick` on the PlayIcon, to avoid the conditional in the container

Reply:
I agree, thats neater